apps/aecore/lib/aecore/utils/persistence.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Aecore.Utils.Persistence do
+  @moduledoc """
+  Store/Restore latest blockchain and chainstate
+  """
+
+  @persistence_table Application.get_env(:aecore, :persistence)[:table]
```

Comment:
"@" constants are calculated on build time, not runtime. Please make a similiar change to commit 950943113c15842ff9256d348553a1a3d75df562